var/spack/repos/builtin/packages/tskit/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Tskit(PythonPackage):
```

Comment:
Any reason not to name this package `py-tskit` like all other Python packages?

Reply:
Hi,
it's a tool that provides a C and a python API so I'm following the policy described at:

https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#choosing-a-package-name 

Ewan

Reply:
The policy you're referring to is for packages that provide an optional Python API, usually enabled by a variant.

Since you're extending the PythonPackage base class, I assume the only thing that gets installed is the Python API, correct? This is similar to the `py-torch` package, which optionally has C++ bindings, but we only install the Python bindings.

I still think it would be better to name this `py-tskit`, although I don't use the package, so the final decision is up to you.